openapi/products/listingBuilder.yaml

```diff
@@ -0,0 +1,214 @@
+      responses:
+        '204':
+          description: No Content
+        '403':
+          description: User Cannot Access Entity
+        '404':
+          description: Business Not Found
+        '406':
+          description: No Listing Product Active
+      tags:
+        - Options
+        - Business Entity Listings
+    parameters: []
+components:
+  schemas:
+    businessLocationReviews:
+      title: Business Entity Listings
+      type: object
+      x-lifecycle:
+        status: trustedTester
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+        A review of an item - for example, of a restaurant, movie, or store.
+      x-tags:
+        - Business Entity Listings
+      properties:
+        name:
+          description: Public facing name of the source
+          type: string
+          example: HotFrog
+        icon:
+          description: Link to icon provided by Vendasta’s media service
+          type: string
+          format: uri
+          example: //www.cdnstyles.com/static/images/icon32/sourceId-10090.png
+        popularity:
+          description: Popularity of the source relative to other sources
+          type: number
+        sourceTypeCode:
+          description: Type of source, business directory, app (Snapchat), GPS app, etc.
+          type: string
+          example: BUSINESS_DIRECTORY
+        syncStatusCode:
+          description: The publishing status provided by the sources. Some sources provide confirmation that your data has been published (synced) other sources only provide confirmation that your data has been received (submitted)
+          type: string
+          example: SYNCED
+        listingScore:
+          description: A score based on the status of your online source
+          type: number
+        maximumlistingScore:
+          description: The maximum score this source can attain
+          type: number
+        url:
+          description: The online listing
+          format: uri
+          type: string
+        syncingError:
+          type: array
+          description: Any errors reported while publishing your information to the source
+          items:
+            type: object
+            properties:
+              errorCode:
+                description: self-explanitory
+                type: string
+                example: STREET_ADDRESS, CATEGORIES
+              errorDetails:
+                description: Human readable details about the error
+                type: string
+        syncingWarning:
+          description: Any warnings reported while publishing your information to the source
+          type: array
+          items:
+            type: object
+            properties:
+              warningCode:
+                description: self-explanitory
```

Comment:
Explain what the partner should do with the value. What are the possible options?